### PR TITLE
Added iscsi package to buildroot iso

### PR DESCRIPTION
**Pull request by aaron-prindle**
Ref: #1174

### CODE REVIEW

File: deploy/iso/minikube-iso/configs/minikube_defconfig [hunk 1]
```diff
@@ -43,3 +43,5 @@ BR2_PACKAGE_GIT=y
 BR2_PACKAGE_LIBCURL=y
 BR2_PACKAGE_CURL=y
 BR2_PACKAGE_RSYNC=y
+BR2_PACKAGE_LIBISCSI=y
+BR2_PACKAGE_BERKELEYDB=y
```
Comment (dlorenc) [inline deploy/iso/minikube-iso/configs/minikube_defconfig]: nit newline

**Comment by fabiand** [comment]
You might also want to change the config to also include the `iscsi_tcp` kmod and deps.

File: deploy/iso/minikube-iso/board/coreos/minikube/linux-4.7_defconfig [hunk 2]
```diff
@@ -420,3 +420,8 @@ CONFIG_SECURITY_NETWORK=y
 CONFIG_SECURITY_SELINUX=y
 CONFIG_SECURITY_SELINUX_BOOTPARAM=y
 CONFIG_SECURITY_SELINUX_DISABLE=y
+CONFIG_INET=y
+CONFIG_BLK_DEV_SD=y
+CONFIG_SCSI_LOWLEVEL=y
+CONFIG_ISCSI_TCP=M
+CONFIG_SCSI_ISCSI_ATTRS=M
```
Comment (r2d4) [inline deploy/iso/minikube-iso/board/coreos/minikube/linux-4.7_defconfig]: nit newline

**Comment by aaron-prindle** [comment]
This currently is not adding the iscsid and iscsiadm binaries to the iso properly, need to getting it building properly.

**Comment by dlorenc** [comment]
Want to close this for now? Or is this still something we need?

**Comment by aaron-prindle** [comment]
I'll close this for now

**Comment by dlorenc** [comment]
Closing, we should keep the issue open but this PR isn't the right approach.

**Pull request closed without merging**